### PR TITLE
Feature/7088 7515 adjust info screen

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
PR for the changed texts on the certificate info screen and the changed legal texts
Ticket:
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7515

Part of this story
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7088

screenshot-dark
screenshot-light

### CODE REVIEW

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings [hunk 1]
```diff
@@ -194,13 +194,13 @@
 
 "HealthCertificate_Info_Legal_headline" = "Datenschutz und Datensicherheit";
 
-"HealthCertificate_Info_Legal_section01" = "​Die Verwendung des digitalen Impfnachweises ist freiwillig. Der Nachweis des vollständigen Impfschutzes kann auch auf andere Weise erbracht werden (z.B. mit dem gelben Impfausweis).";
+"HealthCertificate_Info_Legal_section01" = "​Die Verwendung der digitalen COVID-Zertifikate ist freiwillig. Die Nachweise des vollständigen Impfschutzes oder eines negativen Testergebnisses können auch auf andere Weise erbracht werden.";
```
Comment (NataliaLemmerth) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:197]:
```suggestion
"HealthCertificate_Info_Legal_section01" = "​Die Verwendung der EU digitalen COVID-Zertifikate ist freiwillig. Die Nachweise des vollständigen Impfschutzes oder eines negativen Testergebnisses können auch auf andere Weise erbracht werden.";
```
correct name: **EU digitales COVID-Zertifikat**

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -2532,17 +2532,17 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 /* Health Certificate Information */
 
-"HealthCertificate_Info_Title" = "Impfzertifikat hinzufügen";
+"HealthCertificate_Info_Title" = "Zertifikate hinzufügen";
 
 "HealthCertificate_Info_imageDescription" = "Eine geschützte Person schaut auf ein Smartphone";
 
-"HealthCertificate_Info_description" = "Fügen Sie Ihren digitalen Impfnachweis in der App hinzu. Sobald Ihr Impfschutz vollständig ist, können Sie den QR-Code in der App vorzeigen, um Ihren Impfschutz nachzuweisen.";
+"HealthCertificate_Info_description" = "Fügen Sie ihre digitalen COVID-Zertifikate in der App hinzu, um mit der App Ihren Impfschutz oder ein negatives Testergebnis nachweisen zu können.";
```
Comment (NataliaLemmerth) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2539]:
```suggestion
"HealthCertificate_Info_description" = "Fügen Sie ihre digitalen EU COVID-Zertifikate in der App hinzu, um mit der App Ihren Impfschutz oder ein negatives Testergebnis nachweisen zu können.";
```
correct name: digitales EU COVID-Zertifikat

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -2532,17 +2532,17 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
 
-"HealthCertificate_Info_section01" = "Um ein Impfzertifikat in die App hinzuzufügen, scannen Sie den QR-Code, den Sie bei der Impfung erhalten haben.";
+"HealthCertificate_Info_section01" = "Sie können digitale COVID-Impfzertifikate oder COVID-Testzertifikate in der App hinzufügen.";
```
Comment (NataliaLemmerth) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2541]:
```suggestion
"HealthCertificate_Info_section01" = "Sie können EU digitale COVID-Impfzertifikate oder COVID-Testzertifikate in der App hinzufügen.";
```

**Review by NataliaLemmerth — CHANGES_REQUESTED** [review]
Please correct the name of the EU digitales COVID-Zertifikat and I'll approve the PR. I will adjust Figma.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2539]: Hi. Thanks for the updated texts. Just changed it.

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by nickguendling — APPROVED** [review]

**Pull request merged**